### PR TITLE
btdu: 0.3.1 -> 0.4.0

### PR DESCRIPTION
**Pull request by AtilaSaraiva**
###### Description of changes

Add [deletion](https://github.com/CyberShadow/btdu#deleting) (press d, then ShiftY to confirm)
Add --import, --export, [headless execution](https://github.com/CyberShadow/btdu#headless-mode), and stop conditions
Add --du to output du-compatible output
Improve explanation of ENOENT
Improve error message when given non-top-level subvolume paths
Try to find a TTY even when standard input/output are redirected

###### Things done

- Built on platform(s)
  - [X] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [X] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/tools/misc/btdu/default.nix [hunk 1]
```diff
@@ -1,23 +1,23 @@
 {stdenv, lib, fetchurl, dub, ncurses, ldc, zlib, removeReferencesTo }:
 
 let
-    _d_ae_ver              = "0.0.3100";
+    _d_ae_ver              = "0.0.3141";
```
Comment (thiagokokada) [inline pkgs/tools/misc/btdu/default.nix:4]: Can you explain to me why is this changed and the others are not?

Comment (AtilaSaraiva) [reply pkgs/tools/misc/btdu/default.nix:4]: Thanks for the review! There is no version change for those inputs, so there nothing to change them to:

https://github.com/dlang-community/containers/releases
https://github.com/D-Programming-Deimos/ncurses/tags

**Review by thiagokokada — APPROVED** [review]
